src/containers/AIEvals/AIEvalsRequestAcess/AIEvalsRequestAcess.module.css

```
@@ -0,0 +1,353 @@
+.Container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 40px;
+
+  min-height: calc(100vh - 64px);
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)